Dockerfile

```diff
@@ -22,29 +27,52 @@ RUN mv /var/www/html /var/www/public
 RUN curl -sS https://getcomposer.org/installer \
   | php -- --install-dir=/usr/local/bin --filename=composer
 
+WORKDIR /var/www
```

Comment:
Why are you moving this _before_ installing extensions, instead of keeping it _after_? Is there a rationale for that?

Generally, I tend to set the workdir immediately prior to commands that need to run inside it.

Reply:
The workdir and installing extensions have no correlation.  I put the workdir where it is as a delimiter between default installation and additional extensions.  

The workdir does two things: 1. set the working directory for the next commands 2. set the working directory when you connect to the container.  I'm using the workdir for #2 just like it was done in the original CLOB.  

If workdir were moved to the bottom of the file then all optional extensions would be moved above apache config.  That is why I felt a delimiter was needed; to better group default and non-default configurations.

Reply:
Workdir has been moved to bottom of Dockerfile